lab-lindsay/app/component/game-pad/game-pad.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('angular')
+.module('ngAdventure')
+.component('gamePad', {
+  template: require('./game-pad.html'),
```

Comment:
have you tried passing in a string template in here? ie
template: '<h1> some random stuff to get rendered {{ctrlerName.propertyYouWantToRender}} </h1>' 

angular is pretty rad like that.